lib/Date/Utility.pm

```diff
@@ -1227,25 +1233,29 @@ e.g.
 
 =cut
 
-sub _minus_months {
+sub minus_months {
     my ($self, $months) = @_;
     return $self->_plus_months(-$months);
 }
 
-=head2 _create_trimmed_date
+*_minus_months = \&minus_months;
+
+=head2 create_trimmed_date
 
 Return a valid L<Date::Utility> object whose date part is same with the given year, month and day and time part is not changed. If the day is greater than the max day in that month , then use that max day as the day in the new object.
 
 =cut
 
-sub _create_trimmed_date {
+sub create_trimmed_date {
     my ($self, $year, $month, $day) = @_;
     my $max_day = __PACKAGE__->new(sprintf("%04d-%02d-01", $year, $month))->days_in_month;
     $day = $day < $max_day ? $day : $max_day;
     my $date_string = sprintf("%04d-%02d-%02d %02d:%02d:%02d", $year, $month, $day, $self->hour, $self->minute, $self->second);
     return __PACKAGE__->new($date_string);
 }
 
+*_create_trimmed_date = \&create_trimmed_date;
```

Comment:
Do we have several tests to test those `_*` methods ? If not, should we add some ?

Reply:
added tests